mono_repo/lib/src/commands/github/github_yaml.dart

```diff
@@ -16,8 +16,8 @@ String generateGitHubYml(
   final jobs = rootConfig.expand((config) => config.jobs);
 
   final jobList = Map<String, dynamic>.fromEntries(
```

Comment:
it looks like the `V` type here could be `Map<String, dynamic>`, can't tell for sure

Reply:
fixed. Rebased. Let's get #249 landed and I'll rebase (again)